script.js

```diff
@@ -1,4 +1,154 @@
+var user_lose = 0;
+var user_win = 0;
+var user_draw = 0;
+var current_mode = "waiting";
+var user_name = "";
```

Comment:
It would be best to keep the naming conventions for variables consistent for all of them (i.e. combine words with first word having all lower case and the subsequent words having uppercase first letters.)

Example:
userLose
userWin
userDraw
currentMode
userName